routes/tasks.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+const router = express.Router()
+const tasksController = require('../controllers/tasksController')
+
+/**
+ * @swagger
+ * /tasks/:
+ *   summary: Used to get all the tasks
+ *   responses:
+ *     200:
+ *       description: returns tasks
```

Comment:
Please add schema for task object in `utils/swaggerDefinition.js` and include that schema in 200 response